lib/node_modules/@stdlib/math/base/special/docs/types/index.d.ts

```diff
@@ -2936,7 +2936,7 @@ interface Namespace {
 	*
 	* @example
 	* var v = ns.csc( 3.141592653589793/2.0 );
-	* // returns ~1.0
+	* // returns 1.0
```

Comment:
@Planeshifter Is this correct? The value is exact?

Reply:
@kgryte Yes.